### PR TITLE
Allows charts to be responsive

### PR DESCRIPTION
**Pull request by lancekruegger**
The last few commits have gone in and manually set the width attribute of the SVG element to the chart options width. This doesn't allow the charts to be responsive. It is indicated in lib/base.js line 5 that these charts are to be responsive for resizing. I think I've got this figured out for the most part. I tested with a spark line graph. Let me know your thoughts.

### CODE REVIEW

File: lib/base.js [hunk 1]
```diff
@@ -7,12 +7,12 @@ var base = module.exports = Common.extend({
         if (node[0] === '#') {
           this.element = this._make('div',{
             id: node.replace(/^#/,''),
-            width: '100%'
+            "style": 'width: 100%;'
```
Comment (hasalfred) [inline lib/base.js]: width: 100% should be equivalent to style: 'width:100%', so lets keep it like before here

File: lib/arcBase.js [hunk 2]
```diff
@@ -30,8 +30,9 @@ var arc = module.exports = Base.extend({
         var chart = this.attributes.opts.chart;
         var data = this.attributes.data;
         var svg = this._make('svg',{
-            width: chart.width,
-            height: chart.height,
+            "style": "width: 100%; max-width: "
```
Comment (hasalfred) [inline lib/arcBase.js]: lets have it less code this way.
```
width: '100%',
height: chart.height,
'max-width': chart.width
```

File: lib/base.js (hunk 1 above)
Comment (lancekruegger) [inline lib/base.js]: Hi, with the ```<div>``` element in HTML the width attribute is not actually allowed or valid. This is why I updated it to be ```style="width: 100%;"```. This was one of the main issues that is fixed with the PR. Previously the SVG element set the width of the parent div with its viewBox property as the div would "wrap" it's child content.

File: lib/arcBase.js (hunk 2 above)
Comment (lancekruegger) [inline lib/arcBase.js]: Hi, note previous comment. The ```height=""``` and ```width=""``` attributes are not allowed on ```<div>``` elements. Also, in this case specifically ```max-width=""``` is not a valid attribute either.

File: lib/base.js (hunk 1 above)
Comment (alfredkam) [inline lib/base.js]: oops, your right.  forgot its a div tag :p

File: lib/arcBase.js (hunk 2 above)
Comment (alfredkam) [inline lib/arcBase.js]: just double checked the max-width is indeed not a valid property in svg. But using  ```max-width``` to make it responsive may work with a certain set of browsers (http://thatemil.com/blog/2014/04/06/intrinsic-sizing-of-svg-in-responsive-web-design/) .  May want to take a look at this to do it right. https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/preserveAspectRatio

**Pull request merged**